EventMessageVisualisationStreamer/src/test/java/EventMessage/EventMessageToEventMessagePOJOTest.java

```diff
@@ -21,14 +26,63 @@ public void setUp() throws Exception {
         }
         //create flatbuffer
         FlatBufferBuilder builder = new FlatBufferBuilder();
+
```

Comment:
Perhaps a comment saying that the detector vector needs to be created before starting the event message. Just so we don't try to move it in a year's time.